### ISSUE TITLE
 Response for preflight has invalid HTTP status code 401.

### ISSUE THREAD

**Issue by xxzztt** [Xabaril/JWTSimpleServer#15]
Response for preflight has invalid HTTP status code 401.

**Comment by CarlosLanderas** [comment]
Hello, hace you configured CORS with AddCors/UseCors in the pipeline?

**Comment by CarlosLanderas** [comment]
Possible duplicate of #13 

ping @lurumad 

@xxzztt , I have added a configurePipeline action so you can register CORS in JWTSimpleServer middleware branch.

This is available in version 0.0.4. If this is not working for you please, reopen the issue

```
public void ConfigureServices(IServiceCollection services)
        {
            services.AddSingleton<IAuthenticationProvider, CustomAuthenticationProvider>()
            .AddCors()
            .AddJwtSimpleServer(setup =>
            {
                setup.IssuerSigningKey = SigningKey;
            })
            .AddJwtInMemoryRefreshTokenStore()
            .AddMvcCore().
            AddAuthorization();
        }
        
        public void Configure(IApplicationBuilder app, IHostingEnvironment env)
        {
            app.UseDefaultFiles();
            app.UseStaticFiles();

            app.UseJwtSimpleServer(setup =>
            {
                setup.IssuerSigningKey = SigningKey;
            }, pipeline => {

                pipeline.UseCors(setup =>
                {
                    setup.AllowAnyOrigin().AllowAnyHeader().AllowAnyMethod();
                });
            });

            app.UseMvc();
        }
```
